src/app/feedback/feedback-view.component.ts

```diff
@@ -111,15 +111,16 @@ export class FeedbackViewComponent implements OnInit, OnDestroy {
   }
 
   editTitle(mode) {
-    this.editTitleMode = mode;
+    this.couchService.put(this.dbName + '/' + this.feedback._rev, this.feedback).subscribe(
+      this.editTitleMode = mode);
   }
 
   setTitle() {
     this.couchService.put(this.dbName + '/' + this.feedback._id, this.feedback).subscribe(
       () => {
         this.editTitleMode = false;
```

Comment:
The change should happen here.  I know the function does not list a parameter, but there is one (we can call it `res`) which will include a `rev` property that should overwrite `this.feedback._rev`

Reply:
Awesome. Thank you for the feedback.